constants.py

```diff
@@ -0,0 +1,54 @@
+ULDK_OBREB_DICT_URL = 'https://uldk.gugik.gov.pl/service.php?obiekt=obreb&wynik=nazwa,teryt'
+ULDK_GMINA_DICT_URL = 'https://uldk.gugik.gov.pl/service.php?obiekt=gmina&wynik=gmina,teryt'
+ULDK_POWIAT_DICT_URL = 'https://uldk.gugik.gov.pl/service.php?obiekt=powiat&wynik=powiat,teryt'
+ULDK_WOJEWODZTWO_DICT_URL = 'https://uldk.gugik.gov.pl/service.php?obiekt=wojewodztwo&wynik=wojewodztwo,teryt'
+
+
+DIALOG_MAPPING = {
+    'rdb_dz': {
+        'tab_title': 'Wybór obiektu przez nazwę obrębu i numer działki',
+        'sample_id': '040101_1.0001.1395',
+        'description_label': ' - dla działki: WWPPGG_R.OOOO.[AR_NR].NR_DZ, WWPPGG_R.OOOO.NR_DZ',
+    },
+    'rdb_ob': {
+        'tab_title': 'Wybór obiektu przez nazwę obrębu',
+        'sample_id': '040101_1.0001',
+        'description_label': ' - dla obrębu: WWPPGG_R.OOOO"',
+    },
+    'rdb_gm': {
+        'tab_title': 'Wybór obiektu przez nazwę gminy',
+        'sample_id': '040101_1',
+        'description_label': ' - dla gminy: WWPPGG_R',
+    },
+    'rdb_pw': {
+        'tab_title': 'Wybór obiektu przez nazwę powiatu',
+        'sample_id': '0401',
+        'description_label': ' - dla powiatu: WWPP',
+    },
+    'rdb_wo': {
+        'tab_title': 'Wybór obiektu przez nazwę województwa',
+        'sample_id': '04',
+        'description_label': ' - dla województwa: WW',
+    },
+    'rdb_bu': {
+        'tab_title': 'Wybór obiektu prze identyfikator budynku',
+        'sample_id': '141301_1.0010.713/2.5_BUD',
```

Comment:
Może to sample ID to zmień tak, żeby wszędzie było odniesienie do tego samego terenu, w sensie:
- budynek - 141301_1.0010.713/2.5_BUD
- działka - 141301_1.0010.713/2
- obręb - 141301_1.0010
itd